.gitignore

```diff
@@ -8,3 +8,5 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+**/*.log
+**/.DS_Store
```

Comment:
here
```
*.log
.DS_Store
```
will probs do just fine

Reply:
This may be so in this case, but it can get confusing when handling more complex rules so I think it's fine to leave it like this for now.

Maybe [check this](https://stackoverflow.com/questions/41761128/are-leading-asterisks-redundant-in-gitignore-path-matching-syntax) for a better explanation.